src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -56,7 +66,15 @@ public static Logger getLogger(String name) {
      */
     public static <T> Logger getLogger(Class<T> clazz) {
         requireNonNull(clazz);
-        return getLogger(clazz.getSimpleName());
+        // The BASE_LOGGER_NAME acts as a common prefix for all application loggers,
+        // organizing them based on functionality or module.
+        // The dot (".") serves as a separator, reflecting the package structure and providing a
+        // hierarchical representation.
+
+        // Changing the BASE_LOGGER_NAME affects all related loggers, enabling centralized control
+        // over their configuration and behavior. It allows for easy adjustment of multiple loggers
+        // simultaneously.
```

Comment:
Something is not right. How does changing BASE_LOGGER_NAME (which is just a String) affects other loggers?

Reply:
Seems to be a typo, I meant to refer to base logger with the name of BASE_LOGGER_NAME